roles/firewall/commit/tasks/main.yml

```diff
@@ -3,15 +3,17 @@
   iptables:
       chain: INPUT
       protocol: tcp
-      destination_port: "{{ item }}"
+      destination_port: "{{ item.split(':').1 }}"
+      source: "{{ item.split(':').0 }}"
```

Comment:
Also, thinking about #79, we could make life easier for people by adding the rules to our own chain (say, `opstools_rules`), and then adding a single `-j opstools_rules` to the `INPUT` chain.  This will make it substantially easier to re-order firewall rules.  We may also want to consider inserting that rule at the *top* of the input chain, rather than at the bottom, which may resolve the problem with our rules getting ignored because of a `DROP` or `REJECT` rule.